include/Encode/AlphaEncodingManager.h

```diff
@@ -2,6 +2,43 @@
 
 #include "Encode/EncodingManager.h"
 
+enum class VRCommDataAlphaEncodingCharacter : char {
+	FIN_THUMB = 'A',
+	FIN_INDEX = 'B',
+	FIN_MIDDLE = 'C',
+	FIN_RING = 'D',
+	FIN_PINKY = 'E',
+	JOY_X = 'F',
+	JOY_Y = 'G',
+	JOY_BTN = 'H',
+	BTN_TRG = 'I',
+	BTN_A = 'J',
+	BTN_B = 'K',
+	GES_GRAB = 'L',
+	GES_PINCH = 'M',
+	BTN_MENU = 'N',
+	BTN_CALIB = 'O',
+};
+
+const char VRCommDataAlphaEncodingCharacters[] = {
+	(char)VRCommDataAlphaEncodingCharacter::FIN_THUMB,
+	(char)VRCommDataAlphaEncodingCharacter::FIN_INDEX,
+	(char)VRCommDataAlphaEncodingCharacter::FIN_MIDDLE,
+	(char)VRCommDataAlphaEncodingCharacter::FIN_RING,
+	(char)VRCommDataAlphaEncodingCharacter::FIN_PINKY,
+	(char)VRCommDataAlphaEncodingCharacter::JOY_X,
+	(char)VRCommDataAlphaEncodingCharacter::JOY_Y,
+	(char)VRCommDataAlphaEncodingCharacter::JOY_BTN,
+	(char)VRCommDataAlphaEncodingCharacter::BTN_TRG,
+	(char)VRCommDataAlphaEncodingCharacter::BTN_A,
+	(char)VRCommDataAlphaEncodingCharacter::BTN_B,
+	(char)VRCommDataAlphaEncodingCharacter::GES_GRAB,
+	(char)VRCommDataAlphaEncodingCharacter::GES_PINCH,
+	(char)VRCommDataAlphaEncodingCharacter::BTN_MENU,
+	(char)VRCommDataAlphaEncodingCharacter::BTN_CALIB,
+	(char)0                                             // Turns into a null terminated string
+};
```

Comment:
If these aren't used anywhere else, might be best to move them to the cpp file and make them static? I know they were defined in the header before, but now is probably a good time to move them

Reply:
Fixed with latest commit